app/controllers/users/articles_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Users
+  class ArticlesController < Users::BaseController
+    before_action :authorize_user!, only: %i(edit update)
+
+    expose(:article)
+
+    def new
+    end
+
+    def create
+      article.user = current_user
+      article.save
+      respond_with(article)
+    end
+
+    def edit
+    end
+
+    def update
+      article.save
```

Comment:
Are you sure that this works? As far as i remember on `update` expose not assign attributes, therefore we need to write something like this:

```
def update
  article.update(article_params)
  respond_with(article)
end
```